Add Server-Sent Events BBE (#5533)
ballerina-platform/ballerina-distribution: examples/http-sse-service/http_sse_service.md

# HTTP service - Server-sent events

> Ballerina HTTP services support pushing real-time data to clients using Server-event events (SSE). A stream of type `http:SseEvent` can be returned from service resource methods. This feature automatically handles sending SSE and sets the content type to `text/event-stream` and the transfer encoding to chunked.

Reviewer: ```suggestion
Ballerina HTTP services support pushing real-time data to clients using server-sent events (SSE). A stream of type `http:SseEvent` can be returned from service resource methods. This feature automatically handles sending SSE and sets the content type to `text/event-stream` and the transfer encoding to chunked.
```

> The `http:Service` in Ballerina supports pushing real-time data to clients using Server-Sent Events (SSE). A stream of `http:SseEvent` can be returned from service resource methods. This feature automatically handles sending SSE and sets the content type to `text/event-stream`, and the transfer encoding is automatically set to chunked.

Reviewer: ```suggestion
Ballerina HTTP services support pushing real-time data to clients using Server-event events (SSE). A stream of type `http:SseEvent` can be returned from service resource methods. This feature automatically handles sending SSE and sets the content type to `text/event-stream` and the transfer encoding to chunked.
```
